jaypeasee/whats-cookin
# Feature - Pantry

# Pull Request Template:


#### What's this PR do?
1. Builds out a method for parsing data in the `Pantry` Class.
1. Adds to the user's shopping list when they do not have enough ingredients.
1. Displays ingredients a user needs to cook on click in the modal if they choose that recipe.
1. Displays the user's shopping list.
1. Parses data in the pantry
1. Displays the pantry in a digestable way for the user.

#### Where should the reviewer start?
* Review `pantry-test.js`
* Review `pantry.js`
* Review `scripts.js`
* Review what as been removed from `index.html`

#### How should this be manually tested?
1. Click on the pantry to see that view. Compare that data with whats in the `currentUser`.
1. Click to add a recipe in the modal and see what is needed to cook.
1. After that, click on the shopping list to see a consolidated list.


#### Any background context you want to provide?
Some of this iteration will probably be refactored to go in `pantry.js`.
